src/network_models/copperplate_model.jl

```diff
@@ -6,8 +6,12 @@ function copper_plate(
     time_steps = model_time_steps(optimization_container)
     expressions = get_expression(optimization_container, expression)
     remove_undef!(optimization_container.expressions[expression])
-    constraint =
-        add_cons_container!(optimization_container, :CopperPlateBalance, time_steps)
+    # TODO DT: is it correct to not specify a component type?
```

Comment:
Need a response here

Reply:
Yeah, this doesn't have a component type right now. I am not sure what would be a correct way of specifying this.